css/style.css

```diff
@@ -558,3 +558,284 @@ textarea.form-element {
   border-top-left-radius: 0;
   border-top-right-radius: 0;
 }
+
+/* Add Desktop version Styling */
+
+@media (min-width: 768px) {
+  .menu {
+    display: none;
+  }
+
+  body {
+    position: relative;
+    width: 1440px;
+    height: 916px;
+    left: 0;
+    top: 1px;
+  }
```

Comment:
[Optional] I recommend that you modify the website to remove the overflow on the page. This will improve the user experience of the website.

![image](https://user-images.githubusercontent.com/20554583/200139442-bbaeb681-d8ac-4283-ae6f-6b36d03e5e6c.png)